store/modules/forecastViz.js

```diff
@@ -60,15 +73,18 @@ export default moduleOptions => ({
       }
       this.dispatch('forecastViz/fetch_as_of_truth');
       this.dispatch('forecastViz/fetch_forecasts');
+      state.current_truth = state.temp_current_truth
+      state.as_of_truth = state.temp_as_of_truth
+      state.forecasts = state.temp_forecasts
     },
     set_current_truth(state, new_truth) {
```

Comment:
Let's update the name of this function and the two below to match the variable that's being updated
```suggestion
    set_temp_current_truth(state, new_truth) {
```